build.sbt

```diff
@@ -121,7 +122,7 @@ lazy val root = Project(id="fgbio", base=file("."))
       "org.scala-lang"            %  "scala-reflect" %  scalaVersion.value,
 	    "com.fulcrumgenomics"       %% "dagr-commons"  % "0.1.1-SNAPSHOT",
 	    "com.fulcrumgenomics"       %% "dagr-sopt"     % "0.1.1-SNAPSHOT",
-      "com.github.samtools"       %  "htsjdk"        % "2.1.0" excludeAll(htsjdkAndPicardExcludes: _*),
+      "com.github.samtools"       %  "htsjdk"        % "2.4.1-SNAPSHOT" excludeAll(htsjdkAndPicardExcludes: _*),
```

Comment:
I'll bump this to 2.5 when available later today.  I forget why now, but I did need something not in 2.4.1.